### ISSUE TITLE
Loadout Builder not working correctly

### ISSUE THREAD

**Issue by SableCentaur** [DestinyItemManager/DIM#1222]
I've been dealing with this issue for some time, but haven't had time to really make a post about it.
I have a couple loadouts that are very borderline T12, and I've found that the loadouts feature sometimes has trouble with finding them, and I think it's because there's a value somewhere that is wrong for the bonus node on some armor piece. I think this because if I change the nodes on some of the gear in my 2-5-5 shotgun pulse rifle build, the loadouts feature no longer finds the setup that allows for this build. The issue is illustrated in this image:

http://imgur.com/a/X6Hgf

The left image is what happens when the nodes on my gear are wrong and I'm just searching for a pulse shotgun build with my favorite cloak - no 2-5-5. The top right image is what happens when I show DIM the exact build that CAN get 2-5-5 but are on the wrong nodes, and the bottom right image is what happens when the correct nodes are applied. I'm currently working on trying to figure out which node value is wrong, if that is indeed the issue. If it helps at all and someone just wants to check the values, here's what it is (for my gear at least):
Helmet - 19
Gauntlets - 17
Chest - 25
Boots - 23
Cloak/Ghost - 10
Artifact - 43

Edit: I just tested it. If all of my gear is set to the correct nodes *except* the artifact, DIM can't find the loadout, so there's an issue with artifacts. The only thing that changes between the two pictures here is that I changed the node on the artifact:
http://imgur.com/a/zetLg

**Comment by ericnelson0** [comment]
Let me see if I understand what you're saying: when you change the bonus nodes in-game the values for the artifact change in dim and the builder fails to find the T12 build?

**Comment by SableCentaur** [comment]
That is correct - as long as I have my artifact node set to discipline, the builder can find the loadout, but if it's on intellect it doesn't think that the 2-5-5 with those armor pieces is possible.

**Comment by SunburnedGoose** [comment]
What if we included an option to include borderline builds that are less
than 3-5 stat points from T12

**Comment by SableCentaur** [comment]
Honestly, if it's possible I'd rather just have the issue with the artifact fixed (if possible) or worked around if necessary, I wouldn't use a feature like that all that much. Today I've just been working with this issue by switching the node back and forth if I can't find the loadout I'm hoping to make.

**Comment by ericnelson0** [comment]
Can you actually click the artifact in DIM with the node in both states and report if the stats are the same as in game?

I'd do it myself, but I can't reproduce this problem.

**Comment by SableCentaur** [comment]
http://imgur.com/a/gFt9Z

The stats are correct but the percentage rating is different, which is interesting, probably has to do with the weirdness of int/disc artifacts. Does this help?

**Comment by Doopysmallz** [comment]
I'm running into the same issue, including if you specify a character such as Warlock, the loadout goes under Hunter, if you select Titan the loadout goes under Warlock.  So on and so forth.

**Comment by delphiactual** [comment]
@Doopysmallz, this error has already been fixed, it was introduced unintentionally when setting up internationalization support.

**Comment by tcurtin** [comment]
Still marked open - should be closed?

**Comment by Reiisan** [comment]
It's because DIM is incorrectly using 42 as the maximum bonus stat for an artifact, it's now 43

**Comment by Doopysmallz** [comment]
My apologies. To answer previous questions. I was not logged into gmail. Approx 12 days ago I uninstalled the newest version of chrome to a build from six months ago. Then 6 days ago I upgraded to the newest version. This time I had no issues whatsoever with loadouts or saving loadouts. With two small exceptions. When choosing certain loadouts it transfers all exotic shards as well to that character. Just two days ago it combined/copied all loadouts on warlock and hunter to the same lists. So with each character loadouts for both would appear. 

I cannot confirm at this time if that last issue is still occurring or not.
